docs/templates/additional-content/product-features.mdx

```diff
@@ -0,0 +1,50 @@
+---
+title: <Product name> Information
+description: Dive deeper into <Product Name>.
+---
+
+## Description (Mandatory)
+
+Description of the product including:
+
+  - What it does
+  - Its advantages (summary of most important features)
+  - Use cases
+
+  Optional:
+  - Where does it place on the "compute abstraction" scale?
+  - If it can be combined with other Scaleway products
+
+### Use Cases
+
+| Use Case | Description |
+|---------|-------------|
+| Web Hosting | Deploy and scale public-facing applications |
+| Data Processing | Run batch jobs with flexible compute options |
+| Private Networking | Isolate resources in a secure environment |
+
+## Technical Information (If applicable)
+
+  - How it is built at Scaleway (If available)
+  - Diagrams (If available)
+  - Requirements to use the product (If applicable)
+  - Can it be integrated with other Scaleway products?
+
```

Comment:
I would add a bullet point to document in which AZs the product is/isn't available.